src/targets/node/request.js

```diff
@@ -90,21 +90,21 @@ module.exports = function (source, options) {
   if (source.cookies.length) {
     reqOpts.jar = 'JAR'
 
-    code.push('var jar = request.jar();')
+    code.push('const jar = request.jar();')
 
     var url = source.url
 
     source.cookies.forEach(function (cookie) {
-      code.push('jar.setCookie(request.cookie("%s=%s"), "%s");', encodeURIComponent(cookie.name), encodeURIComponent(cookie.value), url)
+      code.push("jar.setCookie(request.cookie('%s=%s'), '%s');", encodeURIComponent(cookie.name), encodeURIComponent(cookie.value), url)
     })
     code.blank()
   }
 
   if (includeFS) {
-    code.unshift('var fs = require("fs");')
+    code.unshift("const fs = require('fs');")
```

Comment:
There are a few instances where you are changing the quotes from single to double in this file. However its not consistent. Please switch these to all single quotes to maintain consistency with the rest of the code.

Reply:
@reynolek You mean in the generated code, or this target itself?

Reply:
Ugh thats a fair question ... the 2 places you did this do in fact take into account making the generated code use single quotes, and you wrapped them in doubles.

This is probably OK then, as I dont really want to see `\"` in random places either.